### PR TITLE
Firehose with tests

### PR DESCRIPTION
**Pull request by peillis**

### CODE REVIEW

File: lib/ex_aws/config/defaults.ex [hunk 1]
```diff
@@ -17,7 +17,7 @@ defmodule ExAws.Config.Defaults do
       region: "us-east-1",
       port: 80
     },
-    kinesis_firehose: %{
+    firehose: %{
```
Comment (benwilson512) [inline lib/ex_aws/config/defaults.ex]: Although it's kind of verbose I'd prefer if we kept this as `kinesis_firehose` unless we're also going to change the module from `KinesisFirehose` to `Firehose`.

Comment (peillis) [inline lib/ex_aws/config/defaults.ex]: Actually I had to change that in order to get it working. Looks like the service it's just called  firehose on amazon. 
I can change the module to `Firehose` if you prefer that.

Comment (benwilson512) [inline lib/ex_aws/config/defaults.ex]: Yes please, thanks!

Comment (peillis) [inline lib/ex_aws/config/defaults.ex]: ok, done

**Pull request merged**